pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
> We can do something when we properly understand the situation because it is far from clear for me.

The only mystery here is nvidia driver internal functioning. Yes, they are the majority of GL users, but for the moment there's simply already a lot of variables to "understand properly", even leaving aside bugs. 

> Whereas the latter will wait the frame ready signal from your GPU. 

I see, if that's the case though, I don't see why we should even ever prefer normal v-sync anymore? 
Yes, we may say that on the down side, adaptive may still get tearing then. 
But what would happen otherwise? Stuttering, and that's simply it. 
Is there really some case where somebody would prefer it to tearing?

> Whereas standard vsync is either 30fps or 60fps.

Yes, but what about triple buffering?

> And how do we test if we are in exclusive fullscreen or not.

It's exactly the `window!=monitor` part. 
Nvidia exclusive fullscreen behavior for the moment is unknown, thus the additional && vendorid!=nvidia thing I had recommended above, so we can just think to that in another issue, otherwise it will never end.